.github/workflows/Build.yml

```diff
@@ -45,86 +46,125 @@ jobs:
         pytest --cov=cloud_governqance --cov-report=term-missing
         coverage run -m pytest
         coverage report -m
-    - name: 🎥 Publish to coveralls.io
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      run: |
-        pip install coveralls
-        COVERALLS_REPO_TOKEN=${{ secrets.COVERALLS_REPO_TOKEN }} coveralls
-    - name: 🔨 Build and publish distribution 📦 to PyPI
-      env:
-        TWINE_USERNAME: __token__
-        TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
-      run: |
-        pip install setuptools wheel twine
-        python setup.py sdist bdist_wheel
-        twine upload dist/*
-        echo '⌛ Wait till package will be updated in PyPI'
-        # Verfiy and wait till latest cloud-governance version will be updated in Pypi (timeout 900 seconds)
-        timeout=900
-        sleep_time=30
-        expected_version=$(python3 setup.py --version)
-        pip --no-cache-dir install cloud-governance --upgrade
-        build=$(pip freeze | grep cloud-governance | sed 's/==/=/g')
-        actual_version="$(cut -d'=' -f2 <<<"$build")"
-        current_wait_time=0
-        while [[ $current_wait_time < $timeout ]]; do
-            if [[ "$expected_version" = "$actual_version" ]]; then
-              echo 'cloud-governance version:' $expected_version 'was updated in Pypi'
-              break;
-            else
-              # sleep for x seconds
-              echo 'wait' "$((current_wait_time+sleep_time))" 'seconds'
-              sleep $sleep_time
-              current_wait_time="$((current_wait_time+sleep_time))"
-              pip --no-cache-dir install cloud-governance --upgrade
-              build=$(pip freeze | grep cloud-governance | sed 's/==/=/g')
-              actual_version="$(cut -d'=' -f2 <<<"$build")"
-              if (( $current_wait_time == $timeout )); then
-                echo "cloud-governance lastet version did not update in Pypi after 900 seconds - raise failure"
-                exit 1
+
+  build:
+
```

Comment:
extra enter

Reply:
You gave space in the benchmark-runner. That's why I add it. Space is ok, so we can see it is another job.

Reply:
OK